compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.td

```diff
@@ -124,11 +131,11 @@ def EncodingAttr :
     /// Returns a vector with values in `element_types`.
     SmallVector<Type> getElementTypesArray();
 
-    /// Clones an encoding with a new bcast_map
-    EncodingAttr clone(AffineMap bcastMap);
+    /// Clones an encoding with a new indexing map.
+    EncodingAttr clone(AffineMap newIndexingMap);
```

Comment:
I think the comment and maybe the method name can be updated. It looks more like `append a composable map to the "operand_index"-th map list` to me.

Reply:
I renamed it to `cloneWithNewOperandIndexingMap`.